resources/index.php

```diff
@@ -155,7 +155,7 @@
 
 $config = curl_exec($ch);
 
-$file = "/etc/eduVpnProvisioning/localDeviceIds.txt";
+$file = "/var/lib/vpn-provisioning/localDeviceIds.txt";
```

Comment:
Hey Simon, I was testing the revocation of VPN configs on Fedora and I saw that the file localDeviceIds.txt was empty. It seems that Apache can not write the managedId to the localDeviceIds.txt file. It has something to do with the permissions because after following the commands described here it works for my case (https://www.linode.com/community/questions/22429/what-prevents-writing-a-file-with-correct-permissions-and-ownership)

Reply:
Sure, haven't tested this part yet. Will fix permission. Stay tuned.